lib/Controller/PageController.php

```diff
@@ -142,6 +147,27 @@ public function duplicateSession(): Response {
 		return $this->index();
 	}
 
+	/**
+	 * @param string $forUser
+	 * @return ?Room
+	 */
+	protected function createPrivateRoom(string $forUser): ?Room {
+		$user = $this->userManager->get($forUser);
+		if (!$user instanceof IUser) {
+			return null;
+		}
+
+		try {
+			$objectType = '';
+			$objectId = '';
+			$room = $this->roomService->createConversation(Room::TYPE_PUBLIC, "Contact request", $user, $objectType, $objectId);
```

Comment:
you still have local changes with the l10nFactory, can you push them?

Reply:
Ah indeed, done!